LocationList.py

```diff
@@ -1048,12 +1048,12 @@ def shop_address(shop_id, shelf_id):
     ("Forest Temple Well Recovery Heart 1",                          ("Collectable",  0x03,  0x2F, None,                        'Recovery Heart',                        ("Forest Temple", "Vanilla", "Freestanding"))),
     ("Forest Temple Well Recovery Heart 2",                          ("Collectable",  0x03,  0x30, None,                        'Recovery Heart',                        ("Forest Temple", "Vanilla", "Freestanding"))),
     # Forest Temple Vanilla Pots
-    ("Forest Temple Center Room Pot 1",                              ("Collectable",  0x03,  0x20, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
-    ("Forest Temple Center Room Pot 2",                              ("Collectable",  0x03,  0x21, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
-    ("Forest Temple Center Room Pot 3",                              ("Collectable",  0x03,  0x22, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
-    ("Forest Temple Center Room Pot 4",                              ("Collectable",  0x03,  0x23, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
-    ("Forest Temple Center Room Pot 5",                              ("Collectable",  0x03,  0x24, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
-    ("Forest Temple Center Room Pot 6",                              ("Collectable",  0x03,  0x25, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
+    ("Forest Temple Center Room Right Pot 1",                        ("Collectable",  0x03,  0x24, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
+    ("Forest Temple Center Room Right Pot 2",                        ("Collectable",  0x03,  0x20, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
+    ("Forest Temple Center Room Right Pot 3",                        ("Collectable",  0x03,  0x22, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
+    ("Forest Temple Center Room Left Pot 1",                         ("Collectable",  0x03,  0x25, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
+    ("Forest Temple Center Room Left Pot 2",                         ("Collectable",  0x03,  0x23, None,                        'Rupees (20)',                           ("Forest Temple", "Vanilla", "Pot"))),
```

Comment:
Not a huge deal but Left Pot 2 and 3 are swapped if you were to count in order from left to right when facing the pots, which I think you were trying to do? At least you did for the right side pots.

Left side pots based on their flags (the "Default" column in this table), when looking straight at the pots from the middle of the room:
Leftmost pot: 0x25
Middle pot: 0x21
Rightmost pot: 0x23

Reply:
It was supposed to be closest to the entrance of the room to farthest. I dunno how these got mixed up but in some ways I'm surprised I even managed to keep left side on the left side >_> it was kind of confusing lol.

Reply:
Hopefully this is resolved now?